neural_compressor_ort/utils/logger.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2023 Intel Corporation
```

Comment:
Just double checking on the license, maybe steering committee members can answer better: does it need to be relicensed to adopt the ONNX license?

Reply:
License-wise, I believe that's addressed with Apache-2.0. That's also what's stated in the repo requirements: "Must adopt the standard ONNX license(s) [All code projects use the Apache 2.0 license. Documentation repositories must use the Creative Commons License version 4.0.]"

Reply:
ONNX project's copyright line is "Copyright (c) ONNX Project Contributors", not sure if that needs to be added? I'm no expert but good to make sure. @freddychiu @prasanthpul

Reply:
Discussed with Andreas and Prasanth. Original Copyright can stay the same (Intel in this case). Examples of past precedence: https://github.com/onnx/onnx-mlir/blob/733dfac9c388dfe562e3322f0ba24b16e35e7156/src/Builder/FrontendDialectTransformer.cpp#L7

Key requirement for license is Apache-2.0 license which is captured.